src/readme.js

```diff
@@ -0,0 +1,199 @@
+const buildTitleAndBadge = ({ title, license }) => `# ${title}
+![${license}](https://img.shields.io/static/v1?label=${license}&message=License&color=green)`;
+
+const buildTableOfContents = ({ installationIncluded, testIncluded }) => {
+  if (installationIncluded && testIncluded) {
+    return `
+
+## Table of contents
+1. [Description](#description)
+2. [User Story](#user-story)
+3. [User Guidance](#guidance)
+    1. [Usage](#usage)
+    1. [Installation Instructions](#installation)
+    2. [Test Instructions](#test)
+    3. [Contribution Guidelines](#contribution)
+4. [Questions](#questions)
+
+`;
+  }
+  if (installationIncluded && !testIncluded) {
+    return `
+
+## Table of contents
+1. [Description](#description)
+2. [User Story](#user-story)
+3. [User Guidance](#guidance)
+    1. [Usage](#usage)
+    1. [Installation Instructions](#installation)
+    2. [Contribution Guidelines](#contribution)
+4. [Questions](#questions)
+
+`;
+  }
+  if (!installationIncluded && testIncluded) {
+    return `
+
+## Table of contents
+1. [Description](#description)
+2. [User Story](#user-story)
+3. [User Guidance](#guidance)
+    1. [Usage](#usage)
+    1. [Test Instructions](#test)
+    2. [Contribution Guidelines](#contribution)
+4. [Questions](#questions)
+
+`;
+  } else {
+    return `
+
+## Table of contents
+1. [Description](#description)
+2. [User Story](#user-story)
+    1. [Usage](#usage)
+    3. [User Guidance](#guidance)
+    1. [Contribution Guidelines](#contribution)
+4. [Questions](#questions)
+
+`;
+  }
+};
+
+const buildDescription = ({ description, userStory }) => `
+<a id="description"></a>
```

Comment:
These html tags aren't needed in the readme syntax. The links to the table of contents will be made by the use of the `#` syntax.